src/magick-image.ts

```diff
@@ -1328,6 +1330,14 @@ export class MagickImage extends NativeInstance implements IMagickImage {
         this._sigmoidalContrast(true, contrast, midpointOrPercentage, channelsOrUndefined)
     }
 
+    solarize(): void
+    solarize(factorOrNumber: Percentage | number = new Percentage(50)): void {
```

Comment:
Can you change `factorOrNumber` to `numberOrPercentage`? And I like the use of `= new Percentage(50)`. Think I should have used that in some other spots also.